src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -0,0 +1,336 @@
+                                        TargetObject="{Binding ElementName=PingIcon}"
+                                        Value="Clear" />
+                                    <ic:ChangePropertyAction
+                                        PropertyName="Visibility"
+                                        TargetObject="{Binding ElementName=PingIcon}"
+                                        Value="Visible" />
+                                    <ic:ChangePropertyAction
+                                        PropertyName="Foreground"
+                                        TargetObject="{Binding ElementName=PingIcon}"
+                                        Value="{StaticResource SystemFillColorCriticalBrush}" />
+                                </ic:DataTriggerBehavior>
+                                <ic:DataTriggerBehavior
+                                    Binding="{x:Bind Ping, Mode=OneWay}"
+                                    ComparisonCondition="Equal"
+                                    Value="{x:Null}">
+                                    <ic:ChangePropertyAction
+                                        PropertyName="Visibility"
+                                        TargetObject="{Binding ElementName=PingIcon}"
+                                        Value="Collapsed" />
+                                </ic:DataTriggerBehavior>
+                            </i:Interaction.Behaviors>
+                        </SymbolIcon>
+                        <ToggleSwitch
+                            Grid.Column="3"
+                            Width="40"
+                            MinWidth="0"
+                            HorizontalAlignment="Right"
+                            IsOn="{x:Bind Active, Mode=TwoWay}"
+                            OffContent=""
+                            OnContent="" />
+                    </Grid>
+                </DataTemplate>
+            </ListView.ItemTemplate>
+        </ListView>
+
+
+
+        <ContentDialog
+            x:Name="EntryDialog"
+            x:Uid="EntryDialog"
+            IsPrimaryButtonEnabled="{Binding Valid, Mode=TwoWay}"
+            PrimaryButtonStyle="{StaticResource AccentButtonStyle}">
+            <ContentDialog.DataContext>
+                <models:Entry />
+            </ContentDialog.DataContext>
+            <StackPanel
+                MinWidth="480"
+                Margin="0,12,0,0"
+                HorizontalAlignment="Stretch"
+                Spacing="24">
+                <TextBox x:Uid="Address" Text="{Binding Address, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" IsSpellCheckEnabled="False" />
+                <TextBox x:Uid="Hosts" Text="{Binding Hosts, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" IsSpellCheckEnabled="False" />
+                <TextBox x:Uid="Comment" Text="{Binding Comment, Mode=TwoWay}" IsSpellCheckEnabled="False" />
+                <ToggleSwitch x:Uid="Active" IsOn="{Binding Active, Mode=TwoWay}" />
```

Comment:
We should use a checkbox or change the text to "yes" and "no". For me "Active: on" sounds a bit strange.